data/geany.glade

```diff
@@ -713,9 +713,9 @@
   </object>
   <object class="GtkImage" id="image7">
     <property name="visible">True</property>
-    <property name="can_focus">False</property>
+    <property name="can-focus">False</property>
     <property name="stock">gtk-new</property>
-    <property name="icon-size">1</property>
+    <property name="icon_size">1</property>
```

Comment:
Not your fault, but this one is silly (but harmless)

Reply:
Do you know why this happens? It appeared to me that in general, glade tries to use `-` instead of `_` but here it's the other way round.

Reply:
Dunno, maybe a typo in Glade